robocop/checkers/comments.py

```diff
@@ -7,21 +7,31 @@
 from robot.utils import FileReader
 
 from robocop.checkers import RawFileChecker, VisitorChecker
-from robocop.rules import Rule, RuleSeverity
+from robocop.rules import Rule, RuleSeverity, RuleParam
 from robocop.utils import ROBOT_VERSION
 
 rules = {
     "0701": Rule(
+        RuleParam(
+            name="todos",
```

Comment:
I would rename `todos` to `terms` or `markers` to stay consistent. Right now we have named it in many ways:
- todos
- marker
- phrases
- terms
It provides confusion.

Reply:
Hah... Children of many ideas and fixes... Definitely needs consolidation. `marker` or `markers` it will be :smile: